phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -544,7 +556,28 @@ private MetaDataMutationResult updateCache(PName tenantId, String schemaName, St
 
         return result;
     }
-    
+
+    public MetaDataMutationResult updateCache(String schemaName) throws SQLException {
+        return updateCache(schemaName, false);
+    }
+
+    public MetaDataMutationResult updateCache(String schemaName, boolean alwaysHitServer) throws SQLException {
+        long clientTimeStamp = getClientTimeStamp();
+        PSchema schema = null;
+        try {
+            schema = connection.getMetaDataCache().getSchema(new PTableKey(null, schemaName));
```

Comment:
yes, union of all above views will be equal to `select * from t`.
as s1.a , s2.b and s3.c are pointing to same physical table "t" only.